components/atom/label/src/index.scss

```diff
@@ -4,9 +4,10 @@
 $c-atom-label: inherit !default;
 $c-atom-label-optional: $c-gray-light !default;
 $c-atom-label-contrast: $c-white !default;
+$c-atom-label-disabled: inherit !default;
```

Comment:
why disabled color is inherited?

Reply:
I passed this as default color to not break the behaviour of those fields that maybe are already forwarding the disabled property when used, but that do not expect to change the label color. Using inherit as the $c-atom-label I want to assure no field label will get the gray color or any other until is not explicitly specified in the vertical theme.

Reply:
could you add an extra column to the [label type demo](https://sui-components-6un4no4kq-schibstedspain.vercel.app/workbench/atom/label/demo) in order to give them aligned?